packages/visualizations/app/styles/navi-visualizations/visualizations/table.less

```diff
@@ -219,3 +224,50 @@
     font-size: @font-size-xsmall;
   }
 }
+
+.table-header-cell__dropdown-container {
```

Comment:
Since this is a different component, can we have its own less file?

Reply:
We do not have an individual component filer for the dropdown, so it is very matching the pattern how we organize the style file.

Reply:
But it is a different component, we don't have a nested style here? This classname is not nested under the table-widget, so it feels like it belongs in its own file.